### ISSUE TITLE
Strange Cheerio failures

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/citgm#699]
Having consistent failures that are only reproducible when run in CITGM, not when test suite is run by itself 

```
1) cheerio .load should render xml in html() when options.xml = true passed to html():
error:                       | Error: expected '<html><head></head><body><mixedcasetag uppercaseattribute></mixedcasetag></body></html>' to equal '<html><head></head><body><mixedcasetag uppercaseattribute=""></mix
error:                       | at Assertion.assert (node_modules/expect.js/index.js:96:13)
error:                       | at Assertion.be.Assertion.equal (node_modules/expect.js/index.js:216:10)
error:                       | at Assertion.(anonymous function) [as be] (node_modules/expect.js/index.js:69:24)
error:                       | at Context.<anonymous> (test/cheerio.js:375:29)
```

I've skipped it in the lookup but this is likely due to some oddity or bug in CITGM itself

**Comment by BridgeAR** [comment]
That's a test failure in the `cheerio` test suite. See https://github.com/cheeriojs/cheerio/issues/1284

They actually recently fixed it but they did not release the fix so far.

**Comment by richardlau** [comment]
Fix release in cheerio 1.0.0-rc.3: https://github.com/cheeriojs/cheerio/issues/1284#issuecomment-480542151
